expyfun/io/_parse.py

```diff
@@ -73,3 +99,53 @@ def read_tab(fname, group_start='trial_id', group_end='trial_ok'):
             d[key] = [(these_vals[ii], these_times[ii]) for ii in idx]
         data.append(d)
     return data
+    
+    
+    def reconstruct_tracker(fname):
+        # read in raw data
+        raw = read_tab_raw(fname)
```

Comment:
i.e., put `from ..stimuli import TrackerUD, ...` as the first line in this function.

And did you really mean for this function to be nested in `read_tab`?